docs/platform/howto/btar/enable-backup-to-another-region.md

```diff
@@ -0,0 +1,165 @@
+In this instruction, the [Aiven CLI client](/docs/tools/cli) is used to interact
+with Aiven APIs.
+:::
+
+Using CLI, you can enable BTAR for
+
+- [New Aiven service](#new-service-cli) or
+- [Existing Aiven service](#existing-service-cli)
+
+:::note[additional_backup_regions]
+To enable BTAR on an Aiven service, you need to add the ``additional_backup_regions``
+parameter to relevant commands.
+:::
+
+### Create a new service with BTAR via CLI {#new-service-cli}
+
+Use the [avn service create](/docs/tools/cli/service-cli) command to create a new
+service. Include ``additional_backup_regions`` as a parameter to the command and set its
+value to the name of desired cloud region.
+
+```bash
+    avn service create                                            \
+        --service-type service_type_name                          \
+        --cloud cloud_region_name                                 \
+        --plan service_plan_name                                  \
+        -c additional_backup_regions=\[\"name_of_cloud_region\"\] \
+        new_service_name
+```
+
+### Enable BTAR on an existing service via CLI {#existing-service-cli}
+
+Use the [avn service update](/docs/tools/cli/service-cli) command to configure your
+service so that it supports BTAR. Include ``additional_backup_regions`` as a parameter to
+the command and set its value to the name of desired cloud region.
+
+```bash
+    avn service update name_of_existing_service                    \
+        -c additional_backup_regions=[\"name_of_cloud_region\"]
+```
+
+## Back up to another region with API
+
+:::note
+In this instruction, the `curl` command line tool is used to interact with Aiven APIs.
+:::
+
+Using [Aiven APIs](/docs/tools/api), you can enable BTAR for
+
+- [New Aiven service](#new-service-api) or
+- [Existing Aiven service](#existing-service-api)
+
+:::note[additional_backup_regions]
+To enable BTAR on an Aiven service, you need to include the ``additional_backup_regions``
+parameter in relevant calls.
+:::
+
+### Create a new service with BTAR via API {#new-service-api}
+
+Use the [ServiceCreate](https://api.aiven.io/doc/#tag/Service/operation/ServiceCreate) API
+to create a new service with BTAR enabled. When constructing the API request, add the
+``user_config`` object to the request body and nest the ``additional_backup_regions``
+field inside.
+
+```bash
+    curl --request POST                                                    \
+        --url https://api.aiven.io/v1/project/YOUR_PROJECT_NAME/service    \
+        --header 'Authorization: Bearer YOUR_BEARER_TOKEN'                 \
+        --header 'content-type: application/json'                          \
+        --data
+            '{
+            "cloud": "string",
+            "plan": "string",
+            "service_name": "service_2_name",
+            "service_type": "cassandra",
+            "user_config": {
+                "additional_backup_regions": ["cloud-region-name"]
+            }
+        }'
```

Comment:
if you don't mind, could you unindent the block to start under the first ` character? You can select the whole thing and do shift-tab once of twice.

(Same for the other samples.)